src/vscode/superbol-vscode-platform/superbol_commands.ml

```diff
@@ -34,6 +34,22 @@ let command id handler =
   commands := command :: !commands;
   command
 
+let _editor_action_findReferences =
+  command "superbol.editor.action.findReferences" @@ Instance
+    begin fun _instance ~args ->
+      match args with
+      | [arg1; arg2] ->
+        let uri = Uri.parse (Ojs.string_of_js arg1) () in
+        let line = Ojs.get_prop_ascii arg2 "line" |> Ojs.int_of_js in
+        let character = Ojs.get_prop_ascii arg2 "character" |> Ojs.int_of_js in
+        let pos = Position.make ~line ~character in
+        let _ = Commands.executeCommand
+            ~command:"editor.action.findReferences"
+            ~args:[Uri.t_to_js uri; Position.t_to_js pos ]
+        in ()
+      | _ -> ()
```

Comment:
I know we don't do that in existing commands, but it may be worth logging some kind of internal warning here. 

Maybe by appending to the log that is associated with the extension (not sure). To do that you'd add `val Superbol_instance.context: t -> Vscode.ExtensionContext.t` to get the context of `instance`, and then get the URI of the log with `Vscode.ExtensionContext.logUri`. There may be functions to append stuff to a file in `Vscode` or in `Node` directly…

Another way would actually be to create an output channel for the extension itself (we have one for the server, but none for "our" client). That can be done like in the OCaml extension with
```ocaml
let extension_oc : Vscode.OutputChannel.t Lazy.t =
  lazy (Vscode.Window.createOutputChannel ~name:"SuperBOL Studio")
```
then with
```ocaml
let lazy oc = extension_oc in
Vscode.OutputChannel.appendLine oc …
```

Reply:
@NeoKaios Actually that can be delayed for later if you have something else on your plate right now.